projects/media-viewer/src/lib/shared/toolbar.events.service.ts

```diff
@@ -0,0 +1,36 @@
+import {Injectable} from '@angular/core';
+import {BehaviorSubject} from 'rxjs';
+
+// Toolbar Custom-Event Types
+export type HighlightMode = boolean;
+export type DrawMode = boolean;
+
+@Injectable({providedIn: 'root'})
+export class ToolbarEventsService {
+  // Register Observable Subject Events relevant to the Toolbar
+  highlightMode = new BehaviorSubject<HighlightMode>(false);
+  drawMode = new BehaviorSubject<DrawMode>(false);
+  constructor() {}
```

Comment:
can be removed

Reply:
why?

Reply:
or the constructor you mean?

Reply:
yeah, if it doesn't do anything you can just remove  it

Reply:
ok cool, will remove that

Reply:
constructor removed. @linusnorton can you recheck?